pkg/chezmoi/format.go

```diff
@@ -35,22 +40,48 @@ type formatYAML struct{}
 var (
 	// FormatsByName is a map of all FormatsByName by name.
 	FormatsByName = map[string]Format{
-		"json": FormatJSON,
-		"toml": FormatTOML,
-		"yaml": FormatYAML,
+		"hujson": FormatJSONC,
+		"jsonc":  FormatJSONC,
+		"json":   FormatJSON,
+		"toml":   FormatTOML,
+		"yaml":   FormatYAML,
 	}
 
 	// FormatsByExtension is a map of all Formats by extension.
 	FormatsByExtension = map[string]Format{
-		"json": FormatJSON,
-		"toml": FormatTOML,
-		"yaml": FormatYAML,
-		"yml":  FormatYAML,
+		"hujson": FormatJSONC,
```

Comment:
Do you think it's really worth supporting this extension?

Reply:
Fair point, removed extension.